src/NetscriptFunctions/Singularity.ts

```diff
@@ -250,7 +252,12 @@ export function NetscriptSingularity(
       workerScript.running = false; // Prevent workerScript from "finishing execution naturally"
       killWorkerScript(workerScript);
     },
-
+    backupSave: function (): void {
+      return exportGame(player)
+    },
+    backupSaveBonus: function (): boolean {
+      return canGetBonus();
+    },
```

Comment:
These should probably call
```ts
helper.updateDynamicRam("backupSave", getRamCost(player, "backupSave"));
helper.checkSingularityAccess("backupSave")
```
and same for `backupSaveBonus`, to keep in line with the other Singularity functions.

Reply:
ah yep! oops good spot 👍
i've never added a new function that needed ram